lib/pure/collections/hashcommon.nim

```diff
@@ -72,30 +72,59 @@ template getPerturb(t: typed, hc: Hash): UHash =
   # influence the recursion in nextTry earlier rather than later.
   translateBits(cast[uint](hc), numBitsMask)
 
+template findCell(t: typed, hc, mustNextTry): int =
+  let m = maxHash(t)
+  var index: Hash = hc and m
+  var perturb = getPerturb(t, hc)
+  var depth = 0
+
+  ## PARAM: this param can affect performance and could be exposed to users whoe
```

Comment:
Can you explain what does `PARAM: this param` mean? Which `param` is `PARAM`?

Typo: `whoe`.

Reply:
fixed typo, and reworded a bit to clarify:
```
PARAM: `depthThres` can affect performance ...
```
I'm using `PARAM` as a searchable keyword to indicate some tunable parameter (as opposed to some fixed constant); happy to pick any other keyword you suggest for that purpose.